NSubstitute.Analyzers.sln

```diff
@@ -77,14 +73,6 @@ Global
 		{00A000B8-B887-4216-92B2-E810616CE7A8}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{00A000B8-B887-4216-92B2-E810616CE7A8}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{00A000B8-B887-4216-92B2-E810616CE7A8}.Release|Any CPU.Build.0 = Release|Any CPU
-		{4DF03310-98B3-450E-90FE-2EF423A6CC59}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Vsix projects were removed from this file - as this solution is meant to be used on Linux